ui.tests/test-module/specs/formcontainer.spec.js

```diff
@@ -98,6 +118,11 @@ describe('Page/Form Authoring', function () {
                 checkEditDialog(formContainerEditPathSelector);
                 cy.get('.cq-dialog-cancel').click();
             });
+
+            it('data model in container', function () {
```

Comment:
can we add more test cases to cover below scenarios:
- check if data schema save is successful.
- if warning is coming on trying to change schema.
- once schema type is set, we should not be able to edit it.

Reply:
added